ENGAGEHF/Dashboard/ToDo/Views/ToDoRows.swift

```diff
@@ -0,0 +1,48 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct ToDoRow: View {
+    @Binding var task: ToDoItem
+    @State var taskComplete = false
+    
+    var body: some View {
+        HStack {
+            Text(task.task)
```

Comment:
We might want to add a multiple text alignment here and in other sections to account for larger texts and text sizes.